shard.yml

```diff
@@ -10,8 +10,10 @@ development_dependencies:
     branch: master
   ameba:
     github: veelenga/ameba
-    version: 0.8.1
+    version: 0.10.0
+  crelease:
+    github: elorest/CRelease
```

Comment:
what exactly do you need this dependency for? I don't see it being used and there is nothing in the `bin/` directory either.

Reply:
Yeah it seems it doesn't build on install like ameba does. crelease is a helper to manage crystal shard versioning and releases. I can remove it.